packages/sdk/src/functions/crossChain.ts

```diff
@@ -0,0 +1,58 @@
+import { encodeFunctionData, parseAbi } from 'viem'
+import { ERC_6551_DEFAULT } from '../constants'
+
+type CrossChainCallParams = {
+  to: `0x${string}`
+  value: bigint
+  data: `0x${string}`
+  originChainId: number
+  destinationChainId: number
+}
+
+type CrossChainCallData = {
+  to: `0x${string}`
+  value: bigint
+  data: `0x${string}`
+}
+
+function encodeEthToPolygonCall(params: CrossChainCallParams): CrossChainCallData {
+  const fxRoot = '0xfe5e5D361b2ad62c541bAb87C45a0B9B018389a2'
```

Comment:
Not super necessary as these are named well enough but might be worth briefly explaining what they do

Reply:
Good call